frontend/src/metabase/querying/components/TimeseriesChrome/TimeseriesBucketPicker/TimeseriesBucketPicker.unit.spec.tsx

```diff
@@ -87,11 +87,23 @@ describe("TimeseriesBucketPicker", () => {
     expect(getNextBucketName()).toBe("Year");
   });
 
+  it("should allow to show more binning options", async () => {
```

Comment:
Please also add tests for the selected item - when it's within the initial expanded list or not